src/ipc/handler-ipc3.c

```diff
@@ -739,6 +738,7 @@ static int ipc_dma_trace_config(uint32_t header)
 
 error:
 	return err;
+#endif /* CONFIG_SUECREEK || defined __ZEPHYR__ */
```

Comment:
how about to change it like this to make it more readable:
```
#if CONFIG_SUECREEK || defined __ZEPHYR__
static int ipc_dma_trace_config(uint32_t header)
{
	return 0;
}
#else
static int ipc_dma_trace_config(uint32_t header)
{
...
}
#endif
```

Reply:
sure works for me